File: generated-sources/scalaz/mojang-authentication/src/main/scala/com/github/asyncmc/mojang/authentication/scalaz/model/GameProfile.scala
package com.github.asyncmc.mojang.authentication.scalaz.model

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import GameProfile._

case class GameProfile (
  agent: Option[String],
id: Option[String],
name: Option[String],
userId: Option[String],
/* Unix timestamp in milliseconds */
  createdAt: Option[Long],
legacyProfile: Option[Boolean],
suspended: Option[Boolean],
paid: Option[Boolean],
migrated: Option[Boolean],
legacy: Option[Boolean])

object GameProfile {
  import DateTimeCodecs._

  implicit val GameProfileCodecJson: CodecJson[GameProfile] = CodecJson.derive[GameProfile]
  implicit val GameProfileDecoder: EntityDecoder[GameProfile] = jsonOf[GameProfile]
  implicit val GameProfileEncoder: EntityEncoder[GameProfile] = jsonEncoderOf[GameProfile]
}

File: generated-sources/php-slim/mojang-authentication/lib/Model/GameProfile.php
<?php
/**
 * GameProfile
 */
namespace OpenAPIServer\Model;

/**
 * GameProfile
 */
class GameProfile
{
    
    /** @var string $agent */
    private $agent;
    
    /** @var string $id */
    private $id;
    
    /** @var string $name */
    private $name;
    
    /** @var \SplFileObject $userId */
    private $userId;
    
    /** @var int $createdAt Unix timestamp in milliseconds*/
    private $createdAt;
    
    /** @var bool $legacyProfile */
    private $legacyProfile;
    
    /** @var bool $suspended */
    private $suspended;
    
    /** @var bool $paid */
    private $paid;
    
    /** @var bool $migrated */
    private $migrated;
    
    /** @var bool $legacy */
    private $legacy;
}
